code/obj/item/clothing/masks.dm

```diff
@@ -320,6 +320,26 @@
 			return 1
 		return 0
 
+	var/mask_bald = FALSE
```

Comment:
Never define variables in a mix of procs, move these up with the rest of the variables.

Reply:
hopefully fixed this